### ISSUE TITLE
How to add eval dataset into paddleOCR-VL SFT

### ISSUE THREAD

**Issue by jerrywind** [PaddlePaddle/ERNIE#1334]
Hi all,
Recently I'm trying to finetune paddleOCR-VL(0.9B) with SFT following the guide : https://github.com/PaddlePaddle/ERNIE/blob/develop/docs/paddleocr_vl_sft.md .
However, the [example yaml script](https://github.com/PaddlePaddle/ERNIE/blob/develop/examples/configs/PaddleOCR-VL/sft/run_ocr_vl_sft_16k.yaml) seems doesn't support the dataset eval, even I split the example dataset to 2 parts manually.

Is that possible to have eval dataset in this SFT training ? Or is there a replacement to use transformer/peft/trl library to finetune the model? (Due to the flash mask , it looks there is no free lunch from huggingface familiy ....)

BR

**Comment by jerrywind** [comment]
Hi @lugimzzz,
what's the progress ?

**Comment by forBlank** [comment]
Hi, during the training process, checkpoints are saved periodically based on the `save_steps` parameter. We recommend using these checkpoints with PaddleOCR to run inference on your validation set, which allows you to evaluate the training performance.

For more details on PaddleOCR inference usage, please refer to the documentation: https://www.paddleocr.ai/latest/version3.x/pipeline_usage/PaddleOCR-VL.html

**Comment by jerrywind** [comment]
Hi @forBlank ,
Thanks for your comments. 
In an another word, you means currently PaddleOCR-VL doesn't support training with evaluating ?
Some RL fine-tunes like DPO couldn't run with training step together ?
BR

**Comment by forBlank** [comment]
Hi @jerrywind ,
For training with evaluating: It is on our roadmap and will be supported in a future release.
For RL in OCR: The OCR task is a typical perception and recognition problem with a deterministic output. We believe that Supervised Fine-Tuning (SFT) is sufficient for this, so we don’t have plans to support Reinforcement Learning (RL) at this time. Could you please describe the specific requirement or use case you have in mind for using RL?

**Comment by jerrywind** [comment]
Hi @forBlank,
I'm tring to do some research on solving the traditional chinese characters with similar shapes, there is a proper way that to list the similiar shapes list of chars and use RL to tell model about what's right and what's wrong ....
Anyway, I could use other techs to overcome it .
That's for you response.
This ticket could be closed :)
BR
